ror.yml

```diff
@@ -0,0 +1,147 @@
+# Recipe for Ruby on Rails collections
+---
+rh-ror50:
+  name: Ruby on Rails 5.0
+  requires: [rh-ruby23, rh-nodejs4, rh-mongodb32]
+  # install: Copr - Settings - Build options - Click "Edit" on target
+  #   Packages
+  install:
+    - scl-utils-build
+    - rh-ruby23-scldevel
+    - rh-ror50-build
+  packages:
+    - rh-ror50
+    # Packages required by RSpec, Cucumber
+    - rubygem-rspec
+    - rubygem-rspec-core:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-rspec-support:
+        replaced_macros:
```

Comment:
@hhorak yes there is the difference between `replaced_macros` and `macros`. It's not same.
As I would show you Fedora spec file for convenience.

## Case of replaced_macros

Below are a case for `replaced_macros`.
The variable `need_bootstrap_set` has already declared. The modification logic is replacing the value.

http://pkgs.fedoraproject.org/cgit/rpms/rubygem-rspec-core.git/tree/rubygem-rspec-core.spec L12
> %global	need_bootstrap_set	0

```
$ sed -i '/need_bootstrap_set/ s/0/1' something.spec
```

## Case of macros

Below are a case for `macros`. `_with_bootstrap` has not declared in the spec file.
In this case, we need to declare the variable by ourselves for the spec file.

http://pkgs.fedoraproject.org/cgit/rpms/rubygem-actionpack.git/tree/rubygem-actionpack.spec L5
> %{?_with_bootstrap: %global bootstrap 1}

It is like 

```
$ mock --define '%global _with_bootstrap 1' src.rpm file
```
or

```
%global _with_bootstrap 1 <== need to append to the top of the spec file.

EXISTED SPEC FILE
```